ush/hafs_rtofs_download.py

```diff
@@ -0,0 +1,230 @@
+#! /usr/bin/env python3
```

Comment:
We are not supporting RTOFS at this point. Right?

Reply:
Sam wrote some download scripts for it, but it is not supported in the workflow. Since he made the scripts, we can include them here in case RTOFS is supported by CDEPS in the future.